bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/LabelImpl.java

```diff
@@ -78,6 +77,6 @@ public Boolean isVisible() {
      */
     @Override
     public String getValue() {
-        return ComponentUtils.translate(properties.get(PN_TITLE, this.defaultTitle), PN_TITLE, properties, i18n);
+        return ComponentUtils.translate(properties.get(PN_TITLE, "\n"), PN_TITLE, properties, i18n);
```

Comment:
why do we have to use newline character '\n' instead of empty string as default?

Reply:
@barshat7 In order to achieve the same outcome as in AF V1 and as per the comments of @ujjwal5 , we have opted to use line breaks ("\n") instead of an empty string ("").